### ISSUE TITLE
[REVIEW]: popeye: a population receptive field estimation tool

### ISSUE THREAD

**Issue by whedon** [openjournals/joss-reviews#103]
**Submitting author:** @kdesimone (<a href="http://orcid.org/0000-0003-4254-9408">Kevin DeSimone</a>)
**Repository:** <a href="https://github.com/kdesimone/popeye" target ="_blank">https://github.com/kdesimone/popeye</a>
**Version:** v.0.2.0
**Editor:** @cMadan
**Reviewer:** @tknapen
**Archive:** <a href="http://dx.doi.org/10.5281/zenodo.176596" target="_blank">10.5281/zenodo.176596</a>

## Status

[![status](http://joss.theoj.org/papers/053a64ce9fda79e99fe8a703e30e4786/status.svg)](http://joss.theoj.org/papers/053a64ce9fda79e99fe8a703e30e4786)

Status badge code:

```
HTML: <a href="http://joss.theoj.org/papers/053a64ce9fda79e99fe8a703e30e4786"><img src="http://joss.theoj.org/papers/053a64ce9fda79e99fe8a703e30e4786/status.svg"></a>
Markdown: [![status](http://joss.theoj.org/papers/053a64ce9fda79e99fe8a703e30e4786/status.svg)](http://joss.theoj.org/papers/053a64ce9fda79e99fe8a703e30e4786)
```

**Reviewers and authors**:

Please avoid lengthy details of difficulties in the review thread. Instead, please create a new issue in the <a href="https://github.com/kdesimone/popeye" target="_blank">target repository</a> and link to those issues (especially acceptance-blockers) in the review thread below. (For completists: if the target issue tracker is also on GitHub, linking the review thread in the issue or vice versa will create corresponding breadcrumb trails in the link target.)
### Conflict of interest
- [x] As the reviewer I confirm that there are no conflicts of interest for me to review this work (such as being a major contributor to the software).
### General checks
- [x] **Repository:** Is the source code for this software available at the <a target="_blank" href="https://github.com/kdesimone/popeye">repository url</a>?
- [x] **License:** Does the repository contain a plain-text LICENSE file with the contents of an [OSI approved](https://opensource.org/licenses/alphabetical) software license?
- [x] **Version:** Does the release version given match the GitHub release (v.1.0.0)?
- [x] **Authorship:** Has the submitting author (@kdesimone) made major contributions to the software?
### Functionality
- [x] **Installation:** Does installation proceed as outlined in the documentation?
- [x] **Functionality:** Have the functional claims of the software been confirmed?
- [x] **Performance:** Have any performance claims of the software been confirmed?
### Documentation
- [x] **A statement of need:** Do the authors clearly state what problems the software is designed to solve and who the target audience is?
- [x] **Installation instructions:** Is there a clearly-stated list of dependencies? Ideally these should be handled with an automated package management solution.
- [x] **Example usage:** Do the authors include examples of how to use the software (ideally to solve real-world analysis problems).
- [x] **Functionality documentation:** Is the core functionality of the software documented to a satisfactory level (e.g. API method documentation)?
- [x] **Automated tests:** Are there automated tests or manual steps described so that the function of the software can be verified?
- [x] **Community guidelines:** Are there clear guidelines for third parties wishing to 1) Contribute to the software 2) Report issues or problems with the software 3) Seek support
### Software paper

Paper PDF: [10.21105.joss.00103.pdf](https://github.com/openjournals/joss-reviews/files/594749/10.21105.joss.00103.pdf)

- [x] **Authors:** Does the `paper.md` file include a list of authors with their affiliations?
- [x] **A statement of need:** Do the authors clearly state what problems the software is designed to solve and who the target audience is?
- [x] **References:** Do all archival references that should have a DOI list one (e.g. papers, datasets, software)?

**Comment by whedon** [comment]
Hello human, I'm @whedon. I'm here to help you with some common editorial tasks for JOSS.

For a list of things I can do to help you, just type:

```
@whedon commands
```

**Comment by cMadan** [comment]
@tknapen - you should be able to check the boxes in the first post in this issue. For some additional details, please see the reviewer guidelines (http://joss.theoj.org/about#reviewer_guidelines). Otherwise, let me know if you have any questions!

**Comment by cMadan** [comment]
@tknapen - just thought I'd check in, have you had a chance to take a look at the package? Or, do you have any questions about the review process?

**Comment by arfon** [comment]
👋 @tknapen - could you give us an update on when you think you might be able to review this submission?

**Comment by cMadan** [comment]
I just send @tknapen a message via Twitter to also see if he is still able to be a reviewer here.

**Comment by cMadan** [comment]
@tknapen just replied on twitter here: https://twitter.com/Tknapen/status/800829930724610048

So, this should be moving along soon :)

**Comment by tknapen** [comment]
I have ticked all the relevant marks, and opened an issue thread in the Popeye repository pointing to the relevant open issues.

**Comment by cMadan** [comment]
@tknapen - thanks for your help reviewing this, it looks like nearly everything is in order for the paper to be accepted.

@kdesimone - as noted, the version number doesn't match with what was submitted to JOSS. Would it be fair to refer to the current code as v.0.1.0, or do you mean v.0.1.0.dev to be distinct from that?

After that is resolved, I think we just need the code to be archived in a repository such as figshare or zenodo and we'd be all set. Am I missing anything, @arfon?

**Comment by arfon** [comment]
> After that is resolved, I think we just need the code to be archived in a repository such as figshare or zenodo and we'd be all set. Am I missing anything, @arfon?

👍 that's correct @cMadan. Once the review is complete @kdesimone please make an archive of the reviewed software in Zenodo/figshare/other service and update this thread with the DOI of the archive.

**Comment by kdesimone** [comment]
Hi,

Here it is: https://zenodo.org/record/176596

**Comment by cMadan** [comment]
@whedon set 10.5281/zenodo.176596 as archive

**Comment by whedon** [comment]
OK. <a href="http://dx.doi.org/10.5281/zenodo.176596" target="_blank">10.5281/zenodo.176596</a> is the archive.

**Comment by cMadan** [comment]
@kdesimone - can you answer my question about the version number? After that's resolved, we're all set

**Comment by kdesimone** [comment]
I actually incremented the version number to 0.2.0 when I added the DOI tag.

**Comment by cMadan** [comment]
@kdesimone - good to know, thanks!

@arfon - how should I change the version number? (I assume not just editing the first post in this issue thread?) Apart from that, we're all set here!

**Comment by arfon** [comment]
@whedon set 10.5281/zenodo.176596 as archive

**Comment by whedon** [comment]
OK. <a href="http://dx.doi.org/10.5281/zenodo.176596" target="_blank">10.5281/zenodo.176596</a> is the archive.

**Comment by arfon** [comment]
> @arfon - how should I change the version number? (I assume not just editing the first post in this issue thread?) Apart from that, we're all set here!

😁 that's actually all it is. I'll do that now.

**Comment by arfon** [comment]
Sorry I didn't catch this sooner @kdesimone but could you move the references into a `paper.bib` file (and cite them from the Markdown)? We need them to be in a separate file for processing the submission.

**Comment by kdesimone** [comment]
No problem, I added the paper.bib and moved refs out of paper.md.

https://github.com/kdesimone/popeye/blob/master/popeye/paper.bib
https://github.com/kdesimone/popeye/blob/master/popeye/paper.md

**Comment by cMadan** [comment]
@arfon - is there as reason you set the archive DOI again?

**Comment by arfon** [comment]
> @arfon - is there as reason you set the archive DOI again?

This was because we were missing the field at the top of the issue (Archive: Pending) so the update that you requested didn't actually do anything :-)

I've now fixed this in https://github.com/openjournals/joss/commit/8fef34f8946c9d86e9486a7cc50cd2ddad485750

**Comment by kdesimone** [comment]
Are we waiting on me for anything?

**Comment by cMadan** [comment]
@kdesimone, I think you're all set. @arfon needs to compile the revised paper.md and officially accept the submission.

**Comment by kdesimone** [comment]
OK great, just wanted to make sure.  Thanks!

**Comment by arfon** [comment]
Many thanks for the review @tknapen and for editing this submission @cMadan.

@kdesimone - your paper is now accepted into JOSS and your DOI is http://dx.doi.org/10.21105/joss.00103 🚀 🎉 💥
